corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/utils.js

```diff
@@ -178,6 +178,10 @@ hqDefine("cloudcare/js/formplayer/menus/utils", function () {
             }
             if (menuResponse.queryResponse != null) {
                 menuData.sidebarEnabled = true;
+                menuData.title = menuResponse.queryResponse.resultsTitle;
```

Comment:
figuring out the logic here if this line is needed or the change below, however at this moment I don't believe both need to be true. This change is only needed when the query request is sent and this is called to fill the "main" region as a default empty. I expect this to change in the next hour